staticd/static_nht.c

```diff
@@ -18,8 +18,7 @@
 #include "static_nht.h"
 
 static void static_nht_update_path(struct static_path *pn, struct prefix *nhp,
-				   uint32_t nh_num, vrf_id_t nh_vrf_id,
-				   struct vrf *vrf)
+				   uint32_t nh_num, vrf_id_t nh_vrf_id)
```

Comment:
I personally would like this commit broken up into several commits:

a) there is a part of this commit where it is nothing more than rearranging function parameters.  This should be it's own commit
b) there is a part of this commit where it is just abstracting a function call.  This should be it's own commit.
c) there is a part of this commit that is the actual bug fix.  This should be it's own commit.

Reply:
Done.